src/VirtoCommerce.Skyflow.Data/Services/SkyflowClient.cs

```diff
@@ -151,7 +153,18 @@ private async Task<SkyflowBearerTokenResponse> GetBearerTokenInternal(SkyflowSer
         }
         return responseContent;
     }
-
+    private string GetQueryString(object callParams)
+    {
+        var result = string.Empty;
+        if (callParams != null)
+        {
+            var type = callParams.GetType();
+            var props = type.GetProperties();
+            var pairs = props.Select(x => x.Name + "=" + x.GetValue(callParams, null)).ToArray();
```

Comment:
Name and value should be encoded.
Use [Microsoft.AspNetCore.Http.Extensions.QueryBuilder](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.extensions.querybuilder?view=aspnetcore-8.0).